flink-runtime/src/main/java/org/apache/flink/runtime/state/SnapshotResult.java

```diff
@@ -116,6 +116,14 @@ public long getStateSize() {
         return jobManagerOwnedSnapshot != null ? jobManagerOwnedSnapshot.getStateSize() : 0L;
     }
 
+    @Override
+    public <E extends Exception> void accept(StateObjectVisitor<E> visitor) throws E {
+        if (jobManagerOwnedSnapshot != null) {
+            jobManagerOwnedSnapshot.accept(visitor);
+        }
```

Comment:
I think you are right, will add `taskLocalSnapshot` too.

(I wasn't sure because `taskLocalSnapshot` shouldn't take part for example in size calculation. However, this should be a concern of a specific visitor and it would be more clear to follow the contract of `accept()` here of visiting all children)